partner_statement/report/report_statement_common.py

```diff
@@ -66,7 +69,7 @@ def _show_buckets_sql_q1(self, partners, date_end, account_type):
                 WHERE l2.date <= %(date_end)s
             ) as pc ON pc.credit_move_id = l.id
             WHERE l.partner_id IN %(partners)s
-                                AND l.account_internal_type = %(account_type)s
+                                AND l_account_type.type = %(account_type)s
```

Comment:
Why adding extra table? I think the field is still available, at least on `account_account`, isn't it?